prow/prstatus/prstatus.go

```diff
@@ -30,7 +30,8 @@ import (
 	"github.com/sirupsen/logrus"
 	"golang.org/x/oauth2"
 
-	"k8s.io/test-infra/prow/config"
+	configgithub "k8s.io/test-infra/prow/config/github"
+	prowflagutil "k8s.io/test-infra/prow/flagutil"
```

Comment:
I'm aesthetically opposed to having import aliases, especially when there's no conflict with other imports. 

If every import of a new package reassigns the name, that suggests we chose the wrong name. Can we choose a better one that doesn't require this?

Reply:
I don't remember why I gave alias for `prowflagutil`. There is no need for it definitely.

The `config/github` package, as you also suggested, I'll try to move it into `flagutil`.

Reply:
`prowflagutil` alias is removed. And the package `prow/config/github` is moved to be part of `prow/githuboauth`.